src/styles/blocks/container.scss

```diff
@@ -0,0 +1,11 @@
+.container {
+  padding-top: 50px;
+  width: 944px;
```

Comment:
calculate this basing on card width

Reply:
This is last unresolved question. Mentor told me to deploy to find the mistake. Please help me finish it. I'm tired)